packages/dotnet-sdk/src/TeamsFx/Conversation/PagedData.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.TeamsFx.Conversation
+{
+    /// <summary>
+    /// Represents a page of data.
+    /// </summary>
+    public class PagedData<T>
+    {
+        /// <summary>
+        /// Gets or sets the page of items.
+        /// </summary>
+        /// <value>
+        /// The array of items.
+        /// </value>
+        public T[] Data { get; set; } = Array.Empty<T>();
+
+        /// <summary>
+        /// Gets or sets a token for retrieving the next page of results.
+        /// </summary>
+        /// <value>
+        /// The Continuation Token to pass to get the next page of results.
```

Comment:
Better to add something like "*Null or empty token means the page reaches the end*"

Reply:
Updated.